amethyst_assets/src/storage.rs

```diff
@@ -147,7 +147,7 @@ impl<A: Asset> AssetStorage<A> {
                     handle,
                     name,
                     tracker,
-                } => {
+                } => {                    
```

Comment:
what is this?

Reply:
extra whitespace supply for hard times ^^ nah, it'll be cleaned up with the next cargo fmt

Reply:
extra whitespace supply for hard times ^^ nah, it'll be cleaned up with the next cargo fmt

Reply:
The formatting bot would still be nice - maybe just a bot that creates PRs if there are changes.